android-client/Espoo/app/src/main/java/com/example/espoo/view/LoginActivity.kt

```diff
@@ -0,0 +1,92 @@
+package com.example.espoo.view
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.view.View
+import android.widget.Toast
+import com.example.espoo.R
```

Comment:
com.example?

Reply:
I didn't see this. I will refactor to just com.espoo, or do you think we can use another package name?

Reply:
Agree